tests/intern.test.js

```diff
@@ -0,0 +1,30 @@
+const Intern = require("../lib/intern.js")
+
+// test to see if the intern constructor is working correctly
+describe("Intern", () => {
+    it("should create an intern object", () => {
+       const intern = new Intern ("John", 123, "john@email.com", "Birmingham University");
```

Comment:
See comment on manager file about repeated code. Please apply this to the other class files.